### PR TITLE
Replace `unless` with `if not`

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by alexjfisher — COMMENTED** [review]
Why?

**Comment by igalic** [comment]
https://github.com/voxpupuli/puppet-hyperglass/pull/6/files#discussion_r491510280

**Comment by igalic** [comment]
https://github.com/voxpupuli/puppet-hyperglass/pull/6/files#discussion_r491510280

File: manifests/server.pp [hunk 1]
```diff
@@ -41,7 +41,7 @@
     ],
   },
 ) {
-  unless $facts['os']['name'] in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
+  if not $facts['os']['name'] in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
```
Comment (igalic) [inline manifests/server.pp:44]: a more readable form, imo, would be
```puppet 
if $facts['os']['name'] not in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
```
if puppet understands that, that is

Comment (alexjfisher) [reply manifests/server.pp]: I guess it's subjective.  To me, the way it currently is (using `unless`), is the most readable.

Comment (igalic) [reply manifests/server.pp]: ya, i think we should close this pr

**Pull request closed without merging**